src/main/java/com/project/controller/UserController.java

```diff
@@ -25,29 +24,29 @@ public UserController(UserService userService) {
     }
 
     @GetMapping
-    public ResponseEntity<List<User>> getUsers(){
+    public ResponseEntity<List<User>> getUsers() {
         List<User> users = userService.getAll();
         return new ResponseEntity<>(users, HttpStatus.OK);
     }
 
     @GetMapping("/{id}")
     ResponseEntity<User> fetchUser(@PathVariable long id) throws UserNotFoundException {
-        return new ResponseEntity(userService.getById(id), HttpStatus.OK);
+        return new ResponseEntity<>(userService.getById(id), HttpStatus.OK);
     }
 
     @PostMapping
-    ResponseEntity createUser(@RequestBody User newUser) throws UserException {
+    ResponseEntity<HttpStatus> createUser(@RequestBody User newUser) throws UserException {
         User createdUser = userService.addUser(newUser);
```

Comment:
This `createdUser` is never used.

Reply:
removed the unnecessary assignment.
made corresponding changes to ApplicationController class